WhyR/About/proposal/proposal.Rmd

```diff
@@ -0,0 +1,20 @@
+# The proposal
+
+## Overview
+
+This project will outline the differences between S and R to the R community.
+
+## Detail
+Before any work commences we will develop a code of conduct, find new members, and confirm the ownership of all intellectual property. We would like new members who study the history of computing as guides for this research, since we are coders and statisticians.
```

Comment:
```suggestion
Before any work commences, it will be necessary to:  
* develop a code of conduct
* find new members
* confirm the ownership of all intellectual property.  
```